http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -103,6 +103,11 @@ export PROXY_DOCKER_IMAGE=$(proxyDockerImage)
 export CRYPTO_DETECTOR_SERVER_NAME=$(cryptoDetectorServerName)
 export CRYPTO_DETECTOR_DOCKER_IMAGE=$(cryptoDetectorDockerImage)
 export MEM_LIMIT=$(memLimit)
+# Setting the shared docker memory to 50% of the allocated memory limit, converting from byte to mb
```

Comment:
I know I could do it within the scala code in leo, but the list of runtime template values is already sooo long that it felt cleaner this way honestly, but happy to rework if you think otherwise, I just want to get this in front of users ASAP to help unblock them

Reply:
totally fine with this approach 👍

Reply:
hmm... I would honestly prefer to do it in leo. I find whenever I use `awk` it feels elegant to start with, but a year or so down the line its broken by bash's lack of type safety in conjunction with an assumption being changed somewhere else in the application (i.e. `memLimit` changing data types in this case).

Reply:
also, our init scripts aren't tested very well 😬

Reply:
This is a very good point, I'll take a stab at moving the logic out of the init scripts and into Leonardo proper with unit testing tomorrow.

Reply:
@jdcanas I took a stab at it if you want to re-review and merge while I am away, thanks!